src/cargo/core/workspace.rs

```diff
@@ -1645,7 +1646,7 @@ impl WorkspaceRootConfig {
 }
 
 /// A group of fields that are inheritable by members of the workspace
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct InheritableFields {
```

Comment:
Since this is now part of the manifest format, we should move it over there

Reply:
Should I move it to the bottom of `toml/mod.rs`? or below `TomlWorkspace`?

Reply:
Below `TomlWorkspace`